packages/chat/tests/messageUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 2 messages', () => {
+		expect(autoSplitterWithSpaces(testStringNoSpaces.repeat(60), 500)).toStrictEqual([
+			'ThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisi',
+			'satestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatest'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 3 messages', () => {
+		expect(autoSplitterWithSpaces(testStringNoSpaces.repeat(100), 500)).toStrictEqual([
+			'ThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisi',
+			'satestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisates',
+			'tThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatest'
+		]);
+	});
+
+	test('Real use case with 2 messages', () => {
+		expect(autoSplitterWithSpaces(realUseCase, 500)).toStrictEqual([
+			"channel : (es) hola, este es un caso de uso real de alguien que escribe un mensaje realmente largo y, como resultado, tiene que dividirse en muchos mensajes | (pt) olá, este é um caso de uso real de alguém digitando uma mensagem muito longa e tendo que ser dividida em muitas mensagens como resultado | (fr) bonjour c'est un vrai cas d'utilisation de quelqu'un qui tape un très long message et qui doit être divisé en plusieurs messages en conséquence | (ru) привет, это реальный случай, когда кто-то",
+			'набирает очень длинное сообщение, и в результате его приходится разбивать на множество сообщений.'
+		]);
+	});
+
+	test('In a language written from right to left and no messages', () => {
```

Comment:
I'll give you a prize if you tell me how to reduce a non-empty string to include *no* messages ;) probably needs a small description fix.